alertreceiver/overlays/moc/smaug/kustomization.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
```

Comment:
can you also add the namespace to the `kustomization.yaml` : `namespace: opf-alertreceiver`